tests/testprofiles/tests.py

```diff
@@ -32,6 +34,17 @@
 if django.VERSION < (1,7):
     from testprofiles.models import TestProfile
 
+if sys.version_info < (3, 4):
+    # Monkey-patch TestCase to add the assertLogs method introduced in
+    # Python 3.4
+    from unittest2.case import _AssertLogsContext
+
+    class LoggerTestCase(TestCase):
+        def assertLogs(self, logger=None, level=None):
+            return _AssertLogsContext(self, logger, level)
+
+    TestCase = LoggerTestCase
```

Comment:
This change requires users on Python 2.7 to have unittest2 in their python environment, otherwise they will not be able to run the tests. If that's not acceptable, we can figure out something else.

Reply:
No problem adding unittest2 if it's separated and only installed when running tests.

Thank you for such a nice work again, would you please rebase and remove that profile reference since we just dropped support for user profiles?

Reply:
Sure thing!

Reply:
Just to be clear, that means that Python 2 users who install `djangosaml2` cannot immediately run `python manage.py test djangosaml2` after adding it to their installed apps. If they want to run the tests, they need to `pip install djangosaml2[test]`.

Reply:
Yep. As long as tox is configured OK and this is ideally also documented in README, then I'm happy to accept it. Also, it'll allow us to use more test-specific tools in the future (should we need it) once we already have that general support in place.